config/test_cases/test_pull_request_openshift_ansible_tox.yml

```diff
@@ -18,6 +18,20 @@ actions:
     repository: "openshift-ansible"
     script: |-
       tox
+  - type: "script"
```

Comment:
We do not want to pay the extra money to start two VMs on EC2 and pay by the hour there -- no reason not to run this here. If these jobs do not pass today and are not ready to be in the queue, we will have to make this a separate job. However, if they are blocking today there is no scenario under which breaking changes will get into master due to experimentation, so I see no reason not to include them.